docs/source/config/index.rst

```diff
@@ -164,6 +164,13 @@ Config:
     The maximum size (in bytes) of message can be sent during processing.
     Defaults to 64KiB.
 
+.. versionadded:: 0.10
+
+- log_flags (int):
+    Control the prefix for STDOUT logs. Does not control the `LogOutput` plugin.
```

Comment:
Also, was there a reason why you didn't propagate the behaviour down to the LogOutput?